lms/djangoapps/email_marketing/models.py

```diff
@@ -0,0 +1,57 @@
+"""
+Email-marketing-related models.
+"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.models import ConfigurationModel
+
+
+class EmailMarketingConfiguration(ConfigurationModel):
+    """ Email marketing configuration """
+
+    class Meta(object):
+        app_label = "email_marketing"
+
+    sailthru_enabled = models.BooleanField(
+        default=False,
+        help_text=_('Enable Sailthru email marketing support.')
+    )
```

Comment:
I was thinking there might be a need in the future to switch to a different email marketing system, each one having it's own enabled flag.